packages/terra-notification-dialog/src/terra-dev-site/test/notification-dialog/ContentNotificationDialog.test.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Button from 'terra-button';
+import NotificationDialog, { NotificationDialogVariants } from '../../../NotificationDialog';
+
+const clickConfirm = () => {
+  alert('You clicked confirm'); // eslint-disable-line no-alert
+};
+
+class ContentNotificationDialog extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isOpen: false,
+    };
+
+    this.handleOpenModal = this.handleOpenModal.bind(this);
+    this.handleCloseModal = this.handleCloseModal.bind(this);
+  }
+
+  handleOpenModal() {
+    this.setState({ isOpen: true });
+  }
+
+  handleCloseModal() {
+    this.setState({ isOpen: false });
+  }
+
+  render() {
+    return (
+      <div>
```

Comment:
Could replace wrapping `div` with `<React.Fragment>`

Reply:
or shorthand, as i learned recently :D 
`<>`

Reply:
Fixed https://github.com/cerner/terra-framework/pull/899/commits/f95efef89b73c14e91d1b18e9543cd665f7a6191#diff-9108cbcfa912ba73cf01fe6382ac549eR31

Reply:
You're other examples could use the same update.

Reply:
Updated all the examples here https://github.com/cerner/terra-framework/pull/899/commits/35d2d0ec00602f21d5d27309d72d297d8ed9d925